### PR TITLE
[KO] Add `Article_styling_criteria/Formatting`

### PR DESCRIPTION
**Pull request by sylphi**
It contains updates for articles that are affected by this changes (exclude outdated articles).

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Disambiguation/ko.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 ---
 
-# 중의적인 표현
+# 동음이의어
 
-[osu!wiki](/wiki/Main_Page)에 있는 다양한 **중의적인 표현**들을 다루는 문서입니다.
+[osu!wiki](/wiki/Main_Page)에 있는 다양한 **동음이의어**를 다루는 문서입니다.
 
-일반적인 문서에서 제목이 유사한 경우, 반드시 ["For other uses" 햇노트"](/wiki/Article_styling_criteria/Formatting#for-other-uses)에 '중의적인 표현' 페이지 링크를 첨부해야 합니다.
+일반적인 문서에서 제목이 유사한 경우, 반드시 ["다르게도 사용할 수 있습니다" 햇노트](/wiki/Article_styling_criteria/Formatting#다르게도-사용할-수-있습니다)에 동음이의어 페이지 링크를 첨부해야 합니다.
```
Comment (CivilWiki) [inline wiki/Disambiguation/ko.md:9]: must have a For other uses hatnote. 가 "반드시 다르게도 사용할 수 있습니다." 로 번역된 것 같은데, "must"를 "반드시" 라고 번역하는 것 보다는 "필수불가결 혹은 꼭 필요한" 과 같이 다른 방향으로 접근하는 건 어떨까 싶습니다.

"일반적인 문서에서 제목이 유사한 경우 필요에 따라 햇노트를 사용할 수 있습니다"

File: wiki/Disambiguation/ko.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ no_native_review: true
 - [NC](NC)
 - [NM](NM)
 - [오프셋](Offset)
-- [Ouendan](Ouendan)
 - [osu!](osu!)
+- [Ouendan](Ouendan)
```
Comment (CivilWiki) [inline wiki/Disambiguation/ko.md:23]: 이건.. 한국어로 응원단을 사용해야 하나 영문 그대로 사용해야 하나 고민이 많이 들긴 하네요..

**Review by CivilWiki — COMMENTED** [review]
the other translated are all good in my perspective.

File: wiki/Article_styling_criteria/Formatting/ko.md [hunk 3]
```diff
@@ -0,0 +1,903 @@
...
+
+```yaml
+no_native_review: true
+```
+
+### 문서 이름 짓기
+
+*또한 [폴더 이름](#폴더-이름)와 [제목](#제목)도 참조하세요.*
+
+문서의 제목은 단수여야 하며, 문장의 첫 글자를 대문자로 작성해야 합니다. 자세한 내용은 [Wikipedia's naming conventions article](https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(plurals))을 확인하세요.
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Formatting/ko.md:115]: the link here should be translated, the whole thing isn't a name or anything

File: wiki/Article_styling_criteria/Formatting/ko.md [hunk 4]
```diff
@@ -0,0 +1,903 @@
...
+링크 텍스트는 다른 색으로 이미 강조가 되어있으므로, 더 강조할 필요가 없습니다:
+
+```markdown
+[Camellia - OOPARTS](https://cametek.bandcamp.com/track/parts)는 osu!를 위해 특별하게 공식적으로 제작된 곡의 예입니다. osu! World Cup 2020 타이브레이크를 위해 특별히 의뢰되어 제작되었기 때문에 *osu! original*이라고도 합니다.
+```
+
+그러나 링크에 참조된 저작물만이 있는 것이 아니라면 이를 적용하지 않습니다.
+
+```markdown
+[Voltaeyx's beatmap of *TheFatRat - Mayday (feat. Laura Brehm)*](https://osu.ppy.sh/beatmapsets/756794)는 독특한 오버랩 슬라이더 패턴으로 2018년에 상당한 인기를 끌었던 곡입니다.
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Formatting/ko.md:326]: here as well

File: wiki/Article_styling_criteria/Formatting/ko.md [hunk 5]
```diff
@@ -0,0 +1,903 @@
...
+
+```markdown
+> plz enjoy game
+
+—rrtyui
+```
+
+## 주제 나누기
+
+주제 나누기 (수평선이라고도 함)는 드물게 사용해야 합니다. 주제 나누기를 사용하는 경우는 다음과 같습니다 (이에 국한되지는 않음):
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Formatting/ko.md:893]: > (수평선이라고도 함)는

there are a lot of instances where there's no space after parentheses and i suspect that's not correct

File: wiki/Article_styling_criteria/Formatting/ko.md [hunk 6]
```diff
@@ -0,0 +1,903 @@
...
+
+### 기울임체
+
+기울임체은 단일 별표 (`*`)를 사용해야 합니다.
+
+축약형이나 두문자어, 혹은 이니셜이 처음 나오는 경우 기울임체로 표시할 수 있습니다.
+
+기울임체는 강조하거나 가독성의 개선을 위해 사용될 수도 있습니다.
+
+작품이나 비디오 게임의 이름은 기울임체로 표기해야 합니다. osu! — 게임 —은 예외입니다.
```
Comment (Walavouchey) [inline wiki/Article_styling_criteria/Formatting/ko.md:301]: shouldn't it be like this? (if em dashes are even a thing in korean)

```suggestion
작품이나 비디오 게임의 이름은 기울임체로 표기해야 합니다. osu! — 게임 — 은 예외입니다.
```

Comment (CivilWiki) [reply wiki/Article_styling_criteria/Formatting/ko.md:301]: I agree with yours. probably it's a mistake.

File: wiki/Article_styling_criteria/Formatting/ko.md (hunk 3 above)
Comment (CivilWiki) [reply wiki/Article_styling_criteria/Formatting/ko.md:115]: There is no translated wiki page, should find the other ways..

File: wiki/Article_styling_criteria/Formatting/ko.md (hunk 6 above)
Comment (CivilWiki) [reply wiki/Article_styling_criteria/Formatting/ko.md:301]: > I agree with yours. probably it's a mistake.

I realized that it's the correct grammar in Korean after I talked with the author in the private messages about this.

reference: https://kornorms.korean.go.kr/regltn/regltnView.do#a770 (from National Institute of Korean Language)

File: wiki/Article_styling_criteria/Formatting/ko.md (hunk 5 above)
Comment (CivilWiki) [reply wiki/Article_styling_criteria/Formatting/ko.md:893]: no issue on "(수평선이라고도 함)". Rather, We need to fix by deleting the space between "주제 나누기" and "(수평선이라고도 함)"

File: wiki/Article_styling_criteria/Formatting/ko.md [hunk 7]
```diff
@@ -0,0 +1,903 @@
...
+
+예:
+
+```markdown
+osu!api의 첫번째 버전은 2013년 7월 2일에 제공되었으며,[^api-first-usage] 여러 사용자들에게 호평을 받았습니다.[^api-praise] 그로부터 몇 년 후에 출시된 새 버전의 API는 더 많은 기능이 포함되어 있습니다.[^api-v2-2020] 엔드포인트 버전 관리는 웹 API에서는 일반적인 기능입니다.[^web-api]
+
+## 참조
+
+[^api-first-usage]: [peppy의 포럼 스레드 (2013-07-02) "osu!api open beta"](https://osu.ppy.sh/community/forums/posts/2403913)
+[^api-praise]: [Menchi의 포럼 스레드 (2013-11-02) in "osu!api open beta"](https://osu.ppy.sh/community/forums/posts/2662247)
```
[the remark this answers is not in the document]
Comment (CivilWiki) [reply wiki/Article_styling_criteria/Formatting/ko.md:862]: Just delete "in" there.

**Comment by CivilWiki** [comment]
all good

**Review by CivilWiki — APPROVED** [review]
all good!

File: wiki/Article_styling_criteria/Formatting/ko.md (hunk 5 above)
Comment (Walavouchey) [reply wiki/Article_styling_criteria/Formatting/ko.md]: do you then use full-width parentheses, similar to japanese? (`（` and `）`)

idk, use what's correct

File: wiki/Article_styling_criteria/Formatting/ko.md (hunk 3 above)
Comment (Walavouchey) [reply wiki/Article_styling_criteria/Formatting/ko.md:115]: i think you misunderstood. "Wikipedia's naming conventions article" isn't an article title.

to illustrate, in swedish i'd translate it to "Wikipedias namnkonvention för artiklar"

**Review by CivilWiki — APPROVED** [review]
Checked!

File: wiki/Article_styling_criteria/Formatting/ko.md [hunk 8]
```diff
@@ -0,0 +1,903 @@
+# 서식
+
+*작성 표준에 대한 내용은 [Article style criteria/Writing](../Writing)를 참조하세요.*\
+*알림: 이 문서에서는 [RFC 2119](https://tools.ietf.org/html/rfc2119)에 따라 요구사항 수준을 설명합니다.*
```
Comment (TicClick) [inline wiki/Article_styling_criteria/Formatting/ko.md:4]: the rfc 2119 link is meaningless: the memo is in English and explains how English words are to be used and perceived, while the translation is Korean

**Review by TicClick — CHANGES_REQUESTED** [review]
other than this, looks good to me

**Review by TicClick — APPROVED** [review]

**Pull request merged**